apps/services/license-api/src/app/modules/license/clients/firearmLicense/firearmLicenseMapper.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  FirearmProperty,
+  FirearmPropertyList,
+  LicenseInfo,
+} from '@island.is/clients/firearm-license'
+import format from 'date-fns/format'
+import { format as formatNationalId } from 'kennitala'
+
+const formatDateString = (dateTime: string) =>
+  dateTime ? format(new Date(dateTime), 'dd.MM.yyyy') : ''
+
+const parsePropertyForPkpassInput = (properties?: Array<FirearmProperty>) => {
```

Comment:
```suggestion
const formatPropertyForPkpassInput = (properties?: Array<FirearmProperty>) => {
```

Isn't `parse` a misnomer since we're assembling the string for the pass? Format might be better or something else.